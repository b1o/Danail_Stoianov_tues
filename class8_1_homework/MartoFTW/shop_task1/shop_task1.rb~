require "csv"

factorer = ARGV[1]
out_arr = []


	CSV.foreach(ARGV[0]) do |row|

		if row[1] == ARGV[1]
			
			out_arr << row

		end	

	end

	out_arr.each do |elem|
		elem[2] = elem[2].to_f
	end

	out_arr = out_arr.sort_by{|a,b,c,d| c}
	out_arr.reverse!


