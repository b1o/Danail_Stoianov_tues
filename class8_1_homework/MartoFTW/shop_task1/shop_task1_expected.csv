chicken,meskomers,9.3,09/11/2013
cheese,meskomers,3.9,10/11/2013
eggs,meskomers,2.5,01/11/2013
milk,meskomers,0.9,20/11/2013
